doc/launcher.md

```diff
@@ -118,6 +118,8 @@ Spoon provides two out of the shelf decompilers, CFR by default, and Fernflower.
 JarLauncher launcher = new JarLauncher("<path_to_jar>", "<path_to_output_src_dir>", "<path_to_pom>", new FernflowerDecompiler(new File("<path_to_output_src_dir>/src/main/java")));
 ```
 
+:warning: The `JarLauncher` feature (and all features relying on decompilation) are not included in `spoon-core` but in `spoon-bytecode`. If you wnat to use them you should declare a dependency to `spoon-bytecode`.
```

Comment:
typo: wnat-> want. 

Side note: we might need to upgrade the release documentation.

Reply:
type -> thanks, I'll correct ASAP.
Side not -> Totally agree.